### PR TITLE
[add] auth work around & tag in buysearch listing detail

### PR DESCRIPTION
**Pull request by dgrundfest**

### CODE REVIEW

**Comment by vtumrukota** [comment]
Looks good Dave, ill start adding more stuff to the database now

**Comment by josephlin55555** [comment]
too many console logs left in; other than that, it seems pretty good

File: snapmarketapp/app/scripts/profile/profile-controllers.js [hunk 1]
```diff
@@ -6,37 +6,50 @@ angular.module('profile.controllers', ['firebase', 'profile.services'])
...
+    if($rootScope.production){
+      Db.authWithOAuthPopup("facebook", function(error, authData) {
+        if (error) {
+          console.log("Login Failed!", error);
+        } else {
+          console.log("Authenticated successfully with payload");
+
+          //waits until user has been fully loaded, then add user into firebase database
+          users.$loaded().then(function() {
+            users[authData.uid] = Profile(authData);
```
Comment (andyttran) [inline snapmarketapp/app/scripts/profile/profile-controllers.js]: We will need to account for the other places that use Db.auth()

File: snapmarketapp/app/scripts/configuration.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+"use strict";
+
+ angular.module('config', [])
+
+.constant('ENV', {name:'development',apiEndpoint:'http://dev.yoursite.com:10000/',
```
Comment (andyttran) [inline snapmarketapp/app/scripts/configuration.js]: Now that we are including this file, we may want to add the firebase url and reference it in Db controller

File: snapmarketapp/app/scripts/sell/sell-controllers.js [hunk 3]
```diff
@@ -106,11 +106,12 @@ angular.module('sell.controllers', ['ngCordova'])
...
     return {x : tap.x , y : tap.y-headerBarOffset};
   }
-  $scope.position = function(tap){
+  $scope.position = function(item){
+    var offset = 100;
     return {'font-size': '20px',
       position: 'absolute',
-            left: tap.x+'px',
-            top: tap.y+'px'};
+            left: item.x+'px',
```
Comment (andyttran) [inline snapmarketapp/app/scripts/sell/sell-controllers.js]: Lets consider tag repopulation on the buy item detail page. Maybe percents would afford us more flexibility

**Pull request merged**